conf/webpack/common.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const { CleanWebpackPlugin } = require('clean-webpack-plugin');
+const CopyPlugin = require("copy-webpack-plugin");
+
+module.exports = {
+  entry: {
+    application: path.resolve(__dirname, '../..', 'assets', 'javascripts', 'application.js')
+  },
+  plugins: [
+    new CleanWebpackPlugin(),
+    new CopyPlugin({
```

Comment:
Do we need this?
Before I used this, but I think we can add `/assets` folder as StaticPath, then we just use it like we did with Rails
https://beego.me/docs/mvc/view/static.md

Reply:
@hoangmirs Updated https://github.com/Lahphim/go-crawler-challenge/pull/33/commits/5915e9bf0f63201c51de40b56bfde5ec7b3e78a1